docs/lesson04/prepare.md

```diff
@@ -144,7 +206,23 @@ print(result)
 result = queue.dequeue()
 print(result)
 ```
+-->
+
+<!--  C# version -->
+```csharp
+/* --------  Test 1 ----------
+Scenario: Ensure that after adding 3 items to the queue, they can be removed in the proper order
+
+Put (push)  in:  300, 200, 100
+Expected Result out (LIFO queue): 100, 200, 300
+*/
+
+
+```
```

Comment:
Here's an example of this from the prove assignment and the format we might put in the page... We probably want to keep the same text for the scenario (and expected output):
```
        // Test 1
        // Scenario: Create a queue with the following people and turns: Bob (2), Tim (5), Sue (3) and
        //           run until the queue is empty
        // Expected Result: Bob, Tim, Sue, Bob, Tim, Sue, Tim, Sue, Tim, Tim
        Console.WriteLine("Test 1");
        var players = new TakingTurnsQueue();
        players.AddPerson("Bob", 2);
        players.AddPerson("Tim", 5);
        players.AddPerson("Sue", 3);
        // Console.WriteLine(players);    // This can be un-commented out for debug help
        while (players.Length > 0)
            players.GetNextPerson();
        // Defect(s) Found:
```